packages/rollup/test/integration/golden.umd.js.map.sha256_

```diff
@@ -1 +1 @@
-58ccd4b63bbd5ff5e2eb93b2da4ea0ea34dcda1c82b37ae32dd77ed7867f4b9e
\ No newline at end of file
+cbec59b9c2ca7bde7011dbd2b14072ad3d51cbb5c9198cece6cf299071abb090
```

Comment:
The hashes changed but not the files themselves?

Reply:
Input files are now `.js` instead of `.mjs`

Reply:
But I guess they're still es6 since the actual files didn't change 👍